### ISSUE TITLE
CRITICAL: Node v18 drops RIPEMD160

### ISSUE THREAD

**Issue by coolaj86** [dashhive/crowdnode-cli#18]
This is going to be very short-lived unless we update (or replace 😈) `dashcore-lib` due to Node v18+ adopting libssl3 and dropping many deprecated algos, including RIPEMD160, which is used for P2PKH (payment addresses).

Obviously we can't fix DashCore, but we can get a pure JS implementation of RIPEMD160 - though we should probably propose an adoption strategy for secure payment addresses so that we can one day deprecate the current version.

**Comment by coolaj86** [comment]
`dashcore-lib` hasn't been updated to use `package.json.browser`, but instead relies on `crypto-browserify`.

```txt
dashcore-lib
    => crypto-browserify
        => create-hash
            => ripemd160
```

We just need to go find in the code where `rmd160` is used and call those functions directly. Likewise, we could do that for the other algos we need (using `package.json.browser` so that we keep things lightweight and use native implementations).

**Comment by coolaj86** [comment]
Bumped priority because even the average joe (or jojobyte) would run into this problem:
https://github.com/dashhive/crowdnode-cli/issues/16#issuecomment-1176941222

**Comment by coolaj86** [comment]
Fixed via d68470b5c56660bb66ce02356ccba4467e0aa52d in v1.0.7
